test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_String.java

```diff
@@ -585,6 +585,8 @@ public void test_equals() {
 	@Test
 	public void test_equalsIgnoreCase() {
 		AssertJUnit.assertTrue("lc version returned unequal to uc", hwlc.equalsIgnoreCase(hwuc));
+		AssertJUnit.assertTrue("Turkish 'ı' lc version returned unequal to 'İ' uc", "\u0131".equalsIgnoreCase("\u0130"));
+		AssertJUnit.assertTrue("Turkish 'i's with 'j' lc version returned unequal to uc", "\u0131\u0049j".equalsIgnoreCase("\u0069\u0049J"));
```

Comment:
Please add a few more testcases such as `\u0131` & `\u0130` are at middle or end of a string instead of just a single character string.
In addition, could you verify all test cases also pass with `-XX:+CompactStrings`?